trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -0,0 +1,334 @@
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/large_data.png'} alt='Large Data' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Handle large datasets faster
+                </h3>
+                <p>
+                  Excel and Google Sheets aren&apos;t designed for modern data. Insert 1 million rows of data into Excel and you&apos;ll wait 10 minutes every time you update your analysis.
+                </p>
+                <p>
+                  Modern data teams use Python to analyze millions of rows of data in just a couple of seconds. 
+                </p>
+              </div>
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/custom_imports.png'} alt='Custom Import' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Use the full power of Python
+                </h3>
+                <p>
+                  Excel used to be the data collection, analysis, and report presentation tool. Today, modern data teams use Excel primarily as the output of analyses. 
+                </p>
+                <p>
+                  Automating reports in Python lets you use AI and ML to extract insights, connect directly to all of your data sources, and utilize template analyses built by your organization.
+                </p>
+              </div>
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/code.png'} alt='Mito Generated Code' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Reduce bugs and key person risk
+                </h3>
+                <p>
+                  Excel&apos;s greatest strength and weakness are the same -- it hides the analysis logic and encourages you focus on the data. 
+                </p>
+                <p>
+                  Because Excel reports lack structure, its easy to introduce bugs into your report and its nearly impossible to transfer responsibility for a large Excel file to someone else on your team. 
+                </p>
+              </div>
+            </div>
+            
+          </section>
+
+          <section className={pageStyles.background_card}>
+            <div className={classNames('center', spreadsheetAutomationStyles.automate_section_container)}>
+              <h2>
+                Mito is the easiset way to automate with Python and Pandas
+              </h2>
+              <p>
+                Edit a spreadsheet. Generate Python.
```

Comment:
<img width="1190" alt="Screen Shot 2023-07-05 at 2 16 36 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/75f3b02c-b2c8-4910-bc80-242ef97fe8f8">

subtext seems smalle than it should be